clang/lib/Driver/Driver.cpp

```diff
@@ -834,6 +834,9 @@ static bool addSYCLDefaultTriple(Compilation &C,
     if (SYCLTriple.getSubArch() == llvm::Triple::NoSubArch &&
         SYCLTriple.isSPIR())
       return false;
+    if (SYCLTriple.getSubArch() == llvm::Triple::NoSubArch &&
+        SYCLTriple.isSPIRV())
+      return false;
```

Comment:
```suggestion
    if (SYCLTriple.getSubArch() == llvm::Triple::NoSubArch &&
        (SYCLTriple.isSPIR() || SYCLTriple.isSPIRV()))
      return false;
```

Reply:
Thanks. Adding this change as part of upcoming commit.